docs/getting-started/overview.rst

```diff
@@ -1,6 +1,110 @@
+design of the runtime. We wanted to make sure that the entire runtime can be distributed 
+as a set of pieces that developers can pick and choose from, rather than a single, 
+monolithic platform. .NET Core is the first implementation of .NET Platform that will be 
+distributed via `NuGet <http://www.nuget.org/>`_ package manager. 
+
+**Better isolation and versioning** was one of the key influencers to create .NET Core. Today's 
+trends in development practices lead to a very different set than what was available just 
+a couple of years ago. Cloud services are posed to radically change the way developers build, 
+test and run their solutions. Application containers like Docker are introducing new 
+possibilities and opportunities when it comes to server (and services) consolidation. With 
+this in mind, the .NET Core is targeting scenarios where the application, essentially, "takes" 
+the runtime with it. This allows for per-app versioning scenario; the developer needs to worry 
+only about the runtime (a set of packages, as seen in the previous paragraph) that her app 
+is dependent on. 
+
+With all of the above, the .NET Core represents a very generic, cross-platform and modern 
+runtime that is capable of supporting multiple workloads, from cloud services, to desktop 
+applications to CLI tools. 
+
+.NET Framework
+--------------
+
+The .NET Framework is the premier implemetentation of the .NET Platform available for 
+Windows server and client developers. It is a very powerful, very mature framework, with 
+a huge class library (known as the **Framework Class Library**) that supports 
+a wide variety of applications and solutions on Windows. 
+
+The additional stacks built on top the .NET Framework allow developers to build sofware 
+in wide range from console applications to rich web applications. 
+
+`Windows Forms <https://msdn.microsoft.com/en-us/library/dd30h2yb(v=vs.110).aspx>`_ 
+and `Windows Presentation Foundation (WPF) <https://msdn.microsoft.com/en-us/library/ms754130(v=vs.110).aspx>`_ 
+are two User Interface (UI) stacks that allow you to build desktop applications for Windows 
+operating system. Windows Forms' strength is in its rich support for common databinding as well as 
+access to Windows' native user interface controls. WPF, on the other hand, allows you to exercise 
+much more control over the look and feel of your application. Both of them allow for building very 
+rich desktop applications that run on Windows, and you should pick the one that is suited for your 
+use case. 
+
+`Windows Communication Foundation (WCF) <https://msdn.microsoft.com/en-us/library/ms731082(v=vs.110).aspx>`_ 
+is a set of libraries that comprise the middleware services stack on .NET Framework. Its main goal 
+is to allow you to use one set of libraries to create services that can communicate through various 
+supported protocols using various data formats and to be hosted in any process you choose, so they 
```

Comment:
How about:

It allows you to create services that can communicate through various supported protocols using various data formats and that can be hosted in any process you choose. A major feature is that your service is not tied to any particular hosting strategy or approach.